src/xma/include/lib/xmahw_lib.h

```diff
@@ -97,23 +97,13 @@ typedef struct XmaCUCmdObjPrivate
 
 typedef struct XmaHwExecBO
 {
-    xclBufferHandle    handle;
-    char*       data;//execBO size is 4096 in xmahw_hal.cpp
-    bool        in_use;
-    int32_t     cu_index;
-    int32_t     session_id;
-    uint32_t    cu_cmd_id1;//Counter
-    int32_t     cu_cmd_id2;//Random num
-
-  XmaHwExecBO() {
-    in_use = false;
-    handle = NULLBO;
-    data = NULL;
-    cu_index = -1;
-    cu_cmd_id1 = 0;
-    cu_cmd_id2 = 0;
-    session_id = -1;
-  }
+    xrt::kernel xrt_kernel;
+    xrt::run xrt_run;
+    bool        in_use = false;
+    int32_t     cu_index = -1;
```

Comment:
do we need cu_index and xrt::run is anyway going to manage index. Can we remove this?
Samething for cu_cmd_id1 and cu_cmd_id2

Reply:
@chvamshi-xilinx , removed cu_index, session_id as they are not needed but cu_cmd_id1 and cu_cmd_id2 are used in check_all_exec_bo.